script/acceptancetest

```diff
@@ -7,6 +7,12 @@ source `dirname $0`/stackenv
 timeout="60m"
 failed=
 
+if [[ -z "${LOG_DIR}" ]]; then
+    echo "LOG_DIR not set, will create a temp directory"
+    mkdir -p /tmp/devstack-logs
```

Comment:
It can be safety to move mkdir outer the if state, in that case `${LOG_DIR}` included from job vars will be ready in any cases:

```bash
if [[ -z "${LOG_DIR}" ]]; then
    echo "LOG_DIR not set, will set a temp directory"
    LOG_DIR=/tmp/devstack-logs
fi
mkdir -p ${LOG_DIR}
```

Reply:
Good idea!